src/main/java/org/example/model/Response.java

```diff
@@ -14,27 +20,42 @@ public Response(int code, String body, String contentType) {
         this.contentType = contentType;
     }
 
-    public int getCode() {
+    public int getResponseCode() {
```

Comment:
The `Response` is already present in the class name.
Why we should duplicate it?

Reply:
fixed

Reply:
Yes